source/modulo_components/include/modulo_components/ComponentInterface.hpp

```diff
@@ -1155,7 +1184,7 @@ inline void ComponentInterface<NodeT>::send_static_transform(const state_represe
 template<class NodeT>
 inline void
 ComponentInterface<NodeT>::send_static_transforms(const std::vector<state_representation::CartesianPose>& transforms) {
-  this->template publish_transforms(transforms, this->static_tf_broadcaster_);
+  this->template publish_transforms(transforms, this->static_tf_broadcaster_, true);
```

Comment:
This is interesting... does this mean that `send_static_transforms` didn't actually publish static transforms until now? I guess we only used this behaviour in  Python components so far and so didn't notice. Nice catch.

Reply:
No, this only adds "static" to the log string hehe

Reply:
oh ok, that's less bad then 😅 I didn't actually check the function description, I just remembered there was an is_static flag.